src/general-utils/get-version.js

```diff
@@ -0,0 +1,5 @@
+const getVersion = (parser) => {
```

Comment:
Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`. (arrow-body-style)

Reply:
`const getVersion = (parser) => parser.getOSVersion();`?